CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Cromwell Change Log
 
+## 80 Release Notes
+
+### No Longer Supporting wes2cromwell
```

Comment:
I would phrase this as "Direct WES support in Cromwell" and reword to make it sound like a bonus that we're adding all the WES endpoints to the Cromwell API ( "...oh and by the way, we're removing that old wes2cromwell thing at the same time")